game-core/src/main/java/games/strategy/triplea/ui/EconomyPanel.java

```diff
@@ -60,7 +60,7 @@ private void setResourceColumns() {
         }
         statList.add(new ResourceStat(resource));
       }
-      statsResource = statList.toArray(new IStat[statList.size()]);
+      statsResource = statList.toArray(new IStat[0]);
```

Comment:
FWIW, here's the original [reference](https://shipilev.net/blog/2016/arrays-wisdom-ancients/) that provides the justification for the IDE static analysis message in 4ed75fd96.